### PR TITLE
First boot improvements

### PR DESCRIPTION
**Pull request by fonfon**
- one common `firstboot_base.html` template to be used during first boot process
  this only displays the 'help' button in the top (main) navigation bar
- adapted some texts, placements, buttons/links -- choose what you think is more appropriate!
- tiny fixes: button size and focusing of an input field

### CODE REVIEW

File: plinth/modules/first_boot/templates/firstboot_base.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+{% endcomment %}
+
+{% block page_head %}
+  <style type="text/css">
+   .navbar-brand {
+       display: none;
+   }
+  </style>
+{% endblock %}
+
```
Comment (SunilMohanAdapa) [inline plinth/modules/first_boot/templates/firstboot_base.html]: Putting the code to hide logo in base will hide the logo from account creation page which is not intended.

File: plinth/modules/first_boot/templates/firstboot_state0.html [hunk 2]
```diff
@@ -21,39 +21,21 @@
...
     <a href="{% url 'first_boot:state1' %}"
-       class="btn btn-primary btn-large">Next</a>
+       class="btn btn-primary btn-lg">Start Setup</a>
+  </p>
+
+  <p class="text-center" style="font-size: larger">
+    To complete the setup of your {{ cfg.box_name }} you have to provide some
+    basic information.<br>
+    It's really quick, promised!
   </p>
```
Comment (SunilMohanAdapa) [inline plinth/modules/first_boot/templates/firstboot_state0.html]: "It's really quick, promised" sounds a bit too informal.

File: plinth/modules/first_boot/templates/firstboot_state10.html [hunk 3]
```diff
@@ -27,28 +27,15 @@
...
-        <a class="btn btn-lg btn-primary" href="{% url 'networks:index' %}">
-          Go to Networks</a>
-      </div>
-
-      <div class="col-xs-6 text-center">
-        <a class="btn btn-lg btn-primary" href="{% url 'apps:index' %}">
-          Go to Apps</a>
-      </div>
-    </div>
   </div>
```
Comment (SunilMohanAdapa) [inline plinth/modules/first_boot/templates/firstboot_state10.html]: I think giving a specific direction of where to go to from here is a good thing.   The two buttons are helping with that.  I agree with your comment that the two buttons are looking as if they are related to 'Internal' and 'External' network options.  How about we move the "Apps" button just above the "Current Network Configuration" heading?  That way, the two button ugly layout is solved and we are also showing directions on where to go from here.

I believe, that the use of the definite article "The" before FreedomBox here is not correct.

De-emphasizing the links to go to network configuration is fine normally except that users will typically have to connect their devices to FreedomBox next and they need to understand where to connect them.  The networking button then also needs to be emphasized for the changes that may be necessary.

Changing the title to say "Current Network Configuration" is a nice improvement.

If it is okay with you, I will make the necessary changes and commit.

File: plinth/modules/first_boot/templates/firstboot_base.html (hunk 1 above)
Comment (fonfon) [inline plinth/modules/first_boot/templates/firstboot_base.html]: My intention was to remove the link to '/plinth', which I thought to be more confusing than useful during the first boot process.

File: plinth/modules/first_boot/templates/firstboot_state10.html (hunk 3 above)
Comment (fonfon) [inline plinth/modules/first_boot/templates/firstboot_state10.html]: Yes, please go ahead and pick/modify the changes you consider appropriate!

File: plinth/modules/first_boot/templates/firstboot_base.html (hunk 1 above)
Comment (SunilMohanAdapa) [inline plinth/modules/first_boot/templates/firstboot_base.html]: Ahh! that makes sense, I have not thought of that.

File: plinth/modules/first_boot/templates/firstboot_state10.html (hunk 3 above)
Comment (SunilMohanAdapa) [inline plinth/modules/first_boot/templates/firstboot_state10.html]: Thank you, will do.

**Pull request closed without merging**